getting-started/assets/postgres/pg_hba.conf

```diff
@@ -0,0 +1,2 @@
+host        all        all         all                   md5
```

Comment:
I didn't find the need for this personally when running against the provided Postgres Docker image or against cloud providers. Are you using one of these deployment options?

Reply:
I am using what is defined.. I did not think I was using different containers... if I did not do this, the bootstrap failed because connections to the Postgres database were rejected (it was complaining about a missing pg_hba.conf, so I gave it one).  Should the expected container have this?

Reply:
FYI, this is what's being used

edward.hillmann@U-RK427UANA9TF:~/sandbox/polaris-src/polaris$ docker container ls
CONTAINER ID   IMAGE           COMMAND                  CREATED          STATUS                   PORTS                                         NAMES
6da1eaec6c26   postgres:17.5   "docker-entrypoint.s…"   49 minutes ago   Up 7 minutes (healthy)   0.0.0.0:5432->5432/tcp, [::]:5432->5432/tcp   postgres-postgres-1